plugins/logstash/alerta_logstash.py

```diff
@@ -16,8 +16,12 @@
 DEFAULT_LOGSTASH_PORT = 6379
 
 LOGSTASH_HOST = os.environ.get('LOGSTASH_HOST') or app.config.get('LOGSTASH_HOST', DEFAULT_LOGSTASH_HOST)
-LOGSTASH_PORT = os.environ.get('LOGSTASH_PORT') or app.config.get('LOGSTASH_PORT', DEFAULT_LOGSTASH_PORT)
-
+LOGSTASH_PORT = app.config.get('LOGSTASH_PORT', DEFAULT_LOGSTASH_PORT)
+if os.environ.get('LOGSTASH_PORT') is not None:
```

Comment:
This `if` statement should be removed to simplify the code. Similar to this...
https://github.com/alerta/alerta-contrib/blob/master/plugins/prometheus/alerta_prometheus.py#L35